Package.swift

```diff
@@ -128,12 +136,14 @@ let package = Package(
                 .product(name: "AWSLambdaRuntime", package: "swift-aws-lambda-runtime"),
                 .product(name: "AWSLambdaEvents", package: "swift-aws-lambda-events"),
                 .product(name: "AsyncHTTPClient", package: "async-http-client"),
+                .product(name: "OpenAPIGenerator", package: "swift-openapi-generator"),
+                .product(name: "OpenAPIRuntime", package: "swift-openapi-runtime"),
                 .product(name: "SotoSecretsManager", package: "soto"),
                 .product(name: "DiscordBM", package: "DiscordBM"),
                 .target(name: "Extensions"),
             ],
-            path: "./Lambdas/GHHooks",
-            swiftSettings: swiftSettings
+            path: "./Lambdas/GHHooks"
+//            , swiftSettings: swiftSettings
```

Comment:
Why?

Reply:
Because the generators generation don't use existential anys.

Reply:
Good reason. Can I suggest 1) adding existential any-correctness to your draft PR on the generator, and 2) possibly moving the API definitions to a separate "just for the API definitions" target so we minimize the amount of code that loses out on the build setting(s)?

Reply:
For the reference, will be solved by https://github.com/apple/swift-openapi-generator/pull/99.